### ISSUE TITLE
moquitto v2.0.7 `$SYS/broker/heap/#` unavailable

### ISSUE THREAD

**Issue by blusewang** [eclipse/mosquitto#2096]
In Man Page
```
$SYS/broker/heap/current size
The current size of the heap memory in use by mosquitto. Note that this topic may be unavailable depending on compile time options.

```

But in `config.mk` couldn't get any about `time` option. and set `config.mk`
```
WITH_MEMORY_TRACKING:=yes
```

After that, then I subscribe `$SYS/broker/heap/#` got nothing.

**Comment by blusewang** [comment]
Both on FreeBSD and MacOS, compile by `gmake`

**Comment by ralight** [comment]
Thank you, I've now fixed this in the `fixes` branch which will become 2.0.8 soon.
